modules/installation-initializing.adoc

```diff
@@ -507,6 +514,21 @@ subnets:
 - subnet-3
 ----
 endif::aws+restricted[]
+ifdef::azure[]
+.. Define the network and subnets for the VNet to install the cluster under the `platform.azure` field:
++
+[source,yaml]
+----
+networkResourceGroupName: <vnet_resource_group> <1>
+virtualNetwork: <vnet> <2>
+controlPlaneSubnet: <control_plane_subnet> <3>
+computeSubnet: <compute_subnet> <4>
+----
+<1> Replace `<vnet_resource_group>` with the resource group name that contains the existing virtual network (VNet).
+<2> Replace `<vnet>` with the existing virtual network name.
+<3> Replace `<control_plane_subnet>` with the existing subnet name to deploy the control plane machines.
+<4> Replace `<compute_subnet>` with the existing subnet name to deploy compute machines.
+endif::azure[]
```

Comment:
Addressed

Reply:
@arkadeepsen Set the publishing strategy to Internal is mainly functional in IPI to not create public LB and public IP by installer.  For UPI, those resources are created by user or referencing the arm template in this doc. Do you think that arm templates need to be updated for this scenario? or add notes there at least? it is not necessary to create public LB and public IP in Azure Firewall case.

Reply:
reading again the doc, since it adds below statements:
```
By setting this option, you create an internal Ingress Controller and a private load balancer.
```
I think this should be already let user know that only create private lb for this scenario, so maybe no need to add notes or update template (those templates are provided just as an example), please ignore my comments here.